tests/dataloader/test_dataset.py

```diff
@@ -1,7 +1,7 @@
-import matchzoo as mz
-from matchzoo import preprocessors
-from matchzoo.dataloader import Dataset
+import math
 
+import matchzoo as mz
+import numpy as np
```

Comment:
MatchZoo-py imports should be grouped into three groups:
- Standard library imports
- Third-party imports
- matchzoo imports

A space are required between each group.